<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notebook</class>
 <widget class="QWidget" name="Notebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Записная книжка</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите имя контакта</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите номер телефона</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="contactListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>360</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
